src/kuksa_viss_grpc_client.cc

```diff
@@ -0,0 +1,100 @@
+/*
+ *
+ * Copyright 2015 gRPC authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include <iostream>
+#include <fstream>
+#include <memory>
+#include <string>
+
+#include <grpcpp/grpcpp.h>
+
+#include "../build/src/kuksa.grpc.pb.h"
```

Comment:
add `../build/src` als [target_include_directories](https://cmake.org/cmake/help/latest/command/target_include_directories.html). So that only include `#include "kuksa.grpc.pb.h"`

Another suggestion, to put generated file into a directory called `gen` or `generated`, and then use `#include "gen/kuksa.grpc.pb.h"`

Reply:
good idea thank you. I think your first suggestion is the way to go.